README.md

```diff
@@ -337,6 +337,64 @@ are done with the migration.
 
   Modify this column if desired.
 
+### `partitions` ###
+
+- `schema` (type `name`): schema of the table with the partition
+
+- `table_name` (type `name`): name of the related table
+
+- `partition_name` (type `name`): name of the partition
+
+- `orig_name` (type `name`): name of the partition in the remote source
+
+- `type` (type `text`): one of the supported partitioning methods `LIST`,
+  `RANGE` or `HASH`
+
+- `expression` (type `text`): partitioning expression used to define a table's
+  partitioning
+
+- `position` (type `integer`): defines the order of the table partitions (must
+  start at 1)
+
+- `values` (type `text`): for a `RANGE` method, contains the upper bound value,
+  for a `LIST` method, contains a list of comma-separated values
```

Comment:
Only the upper bound for range partitioning? Why not both?
Perhaps that has to do with Oracle's metadata, but I think it is odd and undesirable in general.
At least now I understand why you have the `position` that I criticized before.
Are Oracle partition boundaries inclusive or exclusive (I am too lazy to check right now)?

Reply:
In Oracle and MySQL, it seems to be a catalog pratice to store upper bound only, see documentation below

* ([Oracle](https://docs.oracle.com/en/database/oracle/oracle-database/19/refrn/ALL_TAB_PARTITIONS.html)) ALL_TAB_PARTITIONS.HIGH_VALUE: Partition bound value expression
* ([MySQL](https://dev.mysql.com/doc/refman/8.0/en/information-schema-partitions-table.html)) INFORMATION_SCHEMA.PARTITIONS.PARTITION_DESCRIPTION: For a RANGE partition, it contains the value set in the partition's VALUES LESS THAN clause, which can be either an integer or MAXVALUE.

The upper bound is excluded

* ([Oracle](https://docs.oracle.com/en/database/oracle/oracle-database/21/vldbg/partition-concepts.html#GUID-D6CC12F9-81A5-4E89-872F-024C6161C0E8)) Each partition has a VALUES LESS THAN clause, that specifies a non-inclusive upper bound for the partitions.
* ([MySQL](https://dev.mysql.com/doc/mysql-partitioning-excerpt/8.0/en/partitioning-range.html)) MySQL must be able to evaluate the expression's return value as part of a LESS THAN (<) comparison.

Reply:
With other DBMS, looks like more tedious

* ([SQL Server](https://learn.microsoft.com/en-us/sql/t-sql/statements/create-partition-scheme-transact-sql?view=sql-server-ver16#a-creating-a-partition-scheme-that-maps-each-partition-to-a-different-filegroup)) Bound are build from a list of values where upper bound is included if `RANGE LEFT` is used in creation statement
* ([Sybase](https://infocenter.sybase.com/help/index.jsp?topic=/com.sybase.infocenter.dc32300.1600/doc/html/san1390612300205.html)) Every partition has an inclusive upper bound, which is specified by the values <= clause when the partition is created. 
* ([DB2](https://www.ibm.com/docs/en/db2/11.5?topic=tables-defining-ranges)) INCLUSIVE indicates that all values equal to the specified value are to be included in the data partition containing this boundary.

Reply:
Weird. I checked with list partitioning, and Oracle has the whole list in `HIGH_VALUE`.

I still think that the API should be general, even if both Oracle and MySQL have no support for gaps in the partitioned space. What if someone wants to write a plugin for a database that supports that?
Also, having the lower bound would also make it easier to exclude single partitions from the migration.

Reply:
Thanks for the research! We won't be able to translate inclusive upper partition bounds to PostgreSQL.
But I think we don't have to worry about that. People will just have to live with some rows ending up in a different partition.

Reply:
Hope #12 will handle all of newcoming plugins!